README.md

```diff
@@ -136,5 +136,15 @@ for `CUSTOM_PATH`. The following fixed expansions are supported:
 Expansions are only performed on the values. No expansions are performed on the keys. Note that the JAVA_HOME
 environment cannot be overwritten with this mechanism; use the `javaHome` mechanism in `StaticLauncherConfig` instead.
 
+# go-init
+
+This repository also publishes a binary called `go-init` that supports the commands `start`, `status`, and `stop`, in
+adherence with the
+[Linux Standard Base](http://refspecs.linuxbase.org/LSB_3.1.1/LSB-Core-generic/LSB-Core-generic/iniscrptact.html)
+specification for init scripts. The binary reads configuration from (relative to its working directory)
+`service/bin/launcher-static.yml` and `var/conf/launcher-custom.yml` in the same vein as `go-java-launcher`, and outputs
+to `var/log/startup.log` and other `var/log/${SUB_PROCESS}-startup.log` files. `go-init` does not launch each
```

Comment:
Doesn't this now output to `var/log/${SERVICE_NAME}-startup.log`?

Reply:
Yeah, per our conversation a few days ago, ideally, yes, but afterwards when I looked into it I noticed `ServiceName` is unfortunately not a required field in the config, and sls-packaging doesn't fill it in. I'd rather have the file always be `startup.log` rather than sometimes `my-awesome-service-startup.log` and sometimes `-startup.log`. And making it a required field would require a major version bump :/.

`ServiceName` is luckily still usable as a string key for the primary process in the pidfile and elsewhere in the code, though, since it's just used as a unique key and not read by humans.

Reply:
I was saying that I'm fine with the latter, though input from `infra` people would be nice.

Reply:
I'm fine with leaving this as `startup.log`.
Btw, why would the service name be defined by sls-packaging?
Sls-packaging defines the *product* name, but you could be running multiple instances of the same product with different service names, so it seems like a deployment concern to set this.